opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -519,7 +530,10 @@ def _shutdown():
             return
 
         self._shutdown_event.set()
-        self._daemon_thread.join(timeout=(deadline_ns - time_ns()) / 10**9)
+        if self._daemon_thread:
```

Comment:
This still gives an error when you call shutdown when the interval is `math.inf` because `self._daemon_thread` is created only when the interval > 0 and < inf.